.circleci/config.yml

```diff
@@ -11,28 +11,22 @@ jobs:
     executor: ruby/default
     steps:
       - checkout
-      - restore_cache:
-          keys:
-            - gem-cache-v1-{{ arch }}-{{ checksum "Gemfile.lock" }}
-      - run: bundle -v
-      - ruby/bundle-install
-      - save_cache:
-          key: gem-cache-v1-{{ arch }}-{{ checksum "Gemfile.lock" }}
-          paths:
-            - vendor/bundle
-
+      - ruby/install-deps
   test:
     docker:
       - image: circleci/ruby:2.6.6-node-browsers
         environment:
+          BUNDLE_JOBS: '3'
+          BUNDLE_RETRY: '3'
           POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: password
           RAILS_ENV: test
           TEST_DB: circle_test
-          PHANTOM_JS: phantomjs-2.1.1-linux-x86_64
-      - image: circleci/postgres:9.6-alpine
+      - image: circleci/postgres:13.1
         environment:
           POSTGRES_USER: postgres
-      - image: docker.elastic.co/elasticsearch/elasticsearch:7.11.1
+          POSTGRES_PASSWORD: password
+      - image: docker.elastic.co/elasticsearch/elasticsearch:7.12.0
```

Comment:
Noting for posterity atm Bonsai is on:

```
    "number": "7.2.0",
```

I don't think we'll see any issues with changing minor versions.